package.json

```diff
@@ -68,6 +68,7 @@
     "@nrwl/tao": "13.8.0",
     "@nrwl/web": "13.8.0",
     "@nrwl/workspace": "13.8.0",
+    "@oclif/command": "^1.8.16",
```

Comment:
revert deps

Reply:
I can certainly do it, but I wasn't able to run tests locally without these installed. Are you sure? If so, why?

Reply:
You need to run `npm i` inside packages/cli. We should use npm workspaces and it'll be automatically.